lib/logmsg/logmsg.h

```diff
@@ -229,11 +229,7 @@ typedef struct _LogMessageQueueNode
 {
   struct iv_list_head list;
   LogMessage *msg;
-  union
-  {
-    guint ack_needed:1, embedded:1, flow_control_requested:1;
-    guint __bitfield;
-  };
+  guint ack_needed:1, embedded:1, flow_control_requested:1;
 } LogMessageQueueNode;
```

Comment:
if the branch is still rebased, please squash this into my patch where I introduced bitfield. no need to add two patches with the same intent.